concourse/templates/default.mako

```diff
@@ -231,7 +320,11 @@ else:
 % if job_step.retries():
   attempts: ${job_step.retries()}
 % endif
+% if job_step._on_abort():
+  config: ${f'&{job_variant.variant_name}_{job_step.name}_config'}
```

Comment:
it would be nice to factor-out the calculation of yaml-anchor-name (might add a brief comment explaining the idea, too)